### ISSUE TITLE
@save vs. >>

### ISSUE THREAD

**Issue by asherber** [AnWeber/vscode-httpyac#109]
I was looking for a way to save responses to a file with this extension. The first thing I saw was [output redirection](https://httpyac.github.io/guide/response.html#output-redirection), but I couldn't get it to work.

```
GET https://httpbin.org/get
>> c:/users/me/desktop/test.json
```

I tried also with backslashes instead of slashes, and a blank line after the `GET` line, but no file was saved.

Then I found [Save](https://httpyac.github.io/guide/metaData.html#save), which does work.

Sorry for the stupid question, but is output redirection supposed to work, or has it been obsoleted by Save? Or am I not using output redirection correctly?

**Edit:** Redirection does seem to work if I just give it a filename -- the file is created in the same directory as the .http file. but I can't seem to get it to work with a full path.

**Comment by AnWeber** [comment]
The feature is supposed to work, though I included mainly for [compatibility with Intellij Http Client](https://www.jetbrains.com/help/idea/exploring-http-syntax.html#response-redirect). `# @save` was initially my idea and I had implemented before. They behave differently because the outputRedirection automatically saves the response, `# @save` is a manual step.

I also just ran it successfully for relative paths. Absolute paths does not work. Will look into it.
